src/traces/bar/attributes.js

```diff
@@ -35,8 +42,32 @@ module.exports = {
     y: scatterAttrs.y,
     y0: scatterAttrs.y0,
     dy: scatterAttrs.dy,
+
     text: scatterAttrs.text,
 
+    textposition: {
```

Comment:
@n-riesco what are your thoughts on adding a `textinfo` attribute (similar to the pie) down the road? 

The default value would be `'text'` which corresponds the value linked to the `text` attribute. Another possiblevalues could be `'total'` (i.e. the total in a `stack` barmode).